### PR TITLE
[Facebook Adapter] Add wrapper tests

### PR DESCRIPTION
**Pull request by LCarbonellRod**
## Description
Added some tests for the wrapper class.

**Changes made**
- We deleted the `EncodeNonAsciiCharacters` method since Facebook encodes the non Ascii values natively.

**Added test**
- GetAppSecretProofShouldAlwaysReturnAStringWith64Characters
- VerifySignatureShouldThrowErrorWithNullRequest
- VerifySignatureShouldReturnTrue
- VerifySignatureShouldReturnFalse
- SendMessageAsyncShouldReturnAnEmptyStringWithWrongPath
- SendMessageAsyncShouldThrowAnExceptionWithNullPath
- SendMessageAsyncShouldThrowAnExceptionWithNullPayload

**Added files**
- FacebookWrapperTests.cs
- RequestResponse.json

### CODE REVIEW

File: tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.Tests/FacebookWrapperTests.cs [hunk 1]
```diff
@@ -1,10 +1,94 @@
...
+        }
+
+        [Fact]
+        public void VerifySignatureShouldReturnTrue()
+        {
+            const string requestHash = "SHA1=32ECC29689D860D68A713FF5BA8D7B787C5E8C80";
+            var facebookWrapper = new FacebookClientWrapper(_testOptions);
+            var request = new Mock<HttpRequest>();
+            var stringifyBody = File.ReadAllText(Directory.GetCurrentDirectory() + @"\Files\RequestResponse.json");
+            request.SetupGet(req => req.Headers[It.IsAny<string>()]).Returns(requestHash);
```
Comment (Aliandi) [inline tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.Tests/FacebookWrapperTests.cs]: add a space here

File: tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.Tests/FacebookWrapperTests.cs [hunk 2]
```diff
@@ -1,10 +1,94 @@
...
+            Assert.True(facebookWrapper.VerifySignature(request.Object, stringifyBody));
+        }
+
+        [Fact]
+        public void VerifySignatureShouldReturnFalse()
+        {
+            const string requestHash = "FakeHash";
+            var facebookWrapper = new FacebookClientWrapper(_testOptions);
+            var request = new Mock<HttpRequest>();
+            request.SetupGet(req => req.Headers[It.IsAny<string>()]).Returns(requestHash);
```
Comment (Aliandi) [inline tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.Tests/FacebookWrapperTests.cs]: add a space here

File: tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.Tests/FacebookWrapperTests.cs [hunk 3]
```diff
@@ -1,10 +1,94 @@
...
+            var facebookWrapper = new FacebookClientWrapper(_testOptions);
+            var request = new Mock<HttpRequest>();
+            var stringifyBody = File.ReadAllText(Directory.GetCurrentDirectory() + @"\Files\RequestResponse.json");
+            request.SetupGet(req => req.Headers[It.IsAny<string>()]).Returns(requestHash);
+
+            Assert.True(facebookWrapper.VerifySignature(request.Object, stringifyBody));
+        }
+
+        [Fact]
+        public void VerifySignatureShouldReturnFalse()
```
Comment (Aliandi) [inline tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.Tests/FacebookWrapperTests.cs]: when should it return false?

File: tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.Tests/FacebookWrapperTests.cs [hunk 4]
```diff
@@ -1,10 +1,94 @@
 ﻿using System;
```
Comment (matiasroldan6) [inline tests/Adapters/Microsoft.Bot.Builder.Adapters.Facebook.Tests/FacebookWrapperTests.cs]: Please add Copyright notice

**Pull request merged**